crates/modelardb_server/src/storage/mod.rs

```diff
@@ -624,8 +628,62 @@ impl StorageEngine {
             .map_err(|error| ParquetError::General(error.to_string()))
     }
 
+    /// Create a file name that includes the start timestamp of the first segment in `batch`, the
```

Comment:
I am not opposed to making this change but it seems that we once agreed that we should use "free" functions instead of associated functions like here, whenever possible, according to [this](https://dev.azure.com/ModelarData/ModelarDB%20Implementation/_workitems/edit/180) work item. 

Maybe either delete the work item or change it to make it clear we are doing the opposite.

Reply:
I tried moving methods out of `StorageEngine` but it created a larger PR then moving the two functions into `StorageEngine`. Despite going against what we previously discussed, I tried to refactor as little code as possible to make it easier to review the collection of small changes made in this PR. This way `StorageEngine` is now, as far as I can tell,` consistent and we can make a PR later that only moves the methods that does not take `self` as a parameter into functions and thus should be simple to review since it only changes code without changing it.